WordPressAuthenticator.podspec

```
@@ -1,20 +1,23 @@
 Pod::Spec.new do |s|
   s.name          = "WordPressAuthenticator"
   s.version       = "1.36.0-beta.2"
-  s.summary       = "WordPressAuthenticator implements an easy and elegant way to authenticate your WordPress Apps."
 
+  s.summary       = "WordPressAuthenticator implements an easy and elegant way to authenticate your WordPress Apps."
   s.description   = <<-DESC
                     This framework encapsulates everything required to display the Authentication UI
                     and perform authentication against WordPress.com and WordPress.org sites.
 
-                    Plus: WordPress.com *signup*  is supported.
+                    Plus: WordPress.com *signup* is supported.
                     DESC
 
-  s.homepage      = "https://github.com/wordpress-mobile/WordPressAuthenticator-iOS"
-  s.license       = "GPLv2"
-  s.author        = { "WordPress" => "mobile@automattic.com" }
+  s.homepage      = "http://apps.wordpress.com"
+  s.license       = { :type => "GPLv2", :file => "LICENSE" }
+  s.author        = { "Automattic" => "mobile@automattic.com" }
+  s.social_media_url = "http://twitter.com/WordPressiOS"
```

Was planning to use this same media URL for all our internal pods. I wonder if we have a more suited Twitter account to link there instead?

---

The only other suggestion I have is 

```suggestion
  s.social_media_url = "https://twitter.com/automattic"
```

Less specific than WordPress iOS, but might be more encompassing in the context of a standard for all our libraries, including those that cross or are outside the WordPress boundary.